### PR TITLE
Fix a typo in README-pl

### PR DESCRIPTION
**Pull request by mfilocha**
Remove an extra parenthesis.

### CODE REVIEW

File: README-pl.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ Więcej informacji na temat współpracy przy tworzeniu dokumentacji znajdziesz
 
 # Zasady postępowania
 
-Udział w działaniach społeczności Kubernetes jest regulowany przez [Kodeks postępowania CNCF]((https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/pl.md).
+Udział w działaniach społeczności Kubernetes jest regulowany przez [Kodeks postępowania CNCF](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/pl.md).
```
Comment (kpucynski) [inline README-pl.md:148]:
```suggestion
Udział w działaniach społeczności Kubernetesa jest regulowany przez [Kodeks postępowania CNCF](https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/pl.md).
```

**Review by kpucynski — CHANGES_REQUESTED** [review]

**Review by kpucynski — APPROVED** [review]
/lgtm

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**